docker/k8s/fabric/README.md

```diff
@@ -61,7 +61,8 @@ This serves as a playground to setup k8s cluster for Avalon Fabric proxy model.
 3. To execute a transaction from shell
    ```bash
    cd examples/apps/generic_client
-   ./fabric_generic_client.py -b fabric --workload_id "echo-result" --in_data "Hello"
+   ./fabric_generic_client.py -b fabric --workload_id "echo-result" --in_data "Hello" \
+       --worker_id "singleton-worker-1"
```

Comment:
Update document that how are we getting this worker id as "singleton-worker-1". Otherwise it appears to be some random string. Update in all the places.

Reply:
Ok, i can mention worker_id is same as the one which is configured in config toml file.